### DISCUSSION TITLE
quarkus as (rest|http) lambda on localstack

### DISCUSSION THREAD

**Discussion by nice-monkey** [quarkusio/quarkus#32182] [Community]
I have a lambda (quarkus-amazon-lambda-rest)  compiled natively deployed on aws to serve http requests for an api-gateway. It works great on aws, but I have troubles deploying it on localstack with serverless framework as a lambda to be part of the whole suite of tools I am deploying locally for better developer experience. Anyone has experience doing anything like this?

**Comment by nice-monkey** [comment]
I got it running with localstack, it was a matter of properly setting the `LAMBDA_EXECUTOR` for my localstack env.
I might post a link to a working example here soon.

**Reply by ma-sharifi** [reply]
@nice-monkey 
May I have the link to your example?

**Reply by nice-monkey** [reply]
@ma-sharifi https://github.com/nice-monkey/quarkus-aws/tree/main/api-gateway
